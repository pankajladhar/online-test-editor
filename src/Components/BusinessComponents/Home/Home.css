.Home {
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  text-align: center;
  color: #fff;
  width: 100%; }
  .Home h1 {
    font-size: 60px;
    transition: all .3s ease; }
    @media (max-width: 576px) {
      .Home h1 {
        font-size: 45px;
        line-height: 1;
        margin-bottom: 20px; } }
  .Home h2 {
    font-size: 30px;
    transition: all .3s ease;
    line-height: 1;
    margin-bottom: 40px; }
  .Home a {
    text-decoration: none;
    font-size: 20px;
    padding: 10px 30px;
    line-height: 1;
    background: #2fe09d;
    border-color: #2fe09d;
    color: #fff;
    border-radius: 4px;
    transition: all .3s ease; }
    .Home a:hover {
      background: #25b37d;
      border-color: #25b37d; }
