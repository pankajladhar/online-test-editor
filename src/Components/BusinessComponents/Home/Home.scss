.Home {
  // color: #1b2133;
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  text-align: center;
  color: #fff;

  h1 {
    font-size: 60px;
  }

  h2 {
    font-size: 30px;
    line-height: 1;
    margin-bottom: 40px;
  }

  a {
    text-decoration: none;
    font-size: 20px;
    padding: 10px 30px;
    line-height: 1;
    background: #349284;
    color: #fff;
    border-radius: 4px;
    transition: all .3s ease;
    &:hover {
      color: #1b2133;
      background: #49c7da;
      border-color: #49c7da;
    }
  }
}
