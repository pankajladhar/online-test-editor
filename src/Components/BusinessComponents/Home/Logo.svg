<?xml version="1.0" encoding="UTF-8"?>
<svg width="103px" height="103px" viewBox="0 0 103 103" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="A4" transform="translate(-160.000000, -103.000000)">
            <g id="Group" transform="translate(161.000000, 104.000000)">
                <path d="M50.5,100 C77.8380951,100 100,77.8380951 100,50.5 C100,23.1619049 77.8380951,1 50.5,1 C23.1619049,1 1,23.1619049 1,50.5" id="Oval-Copy" stroke="#FFF" stroke-width="4"></path>
                <path d="M50,90 C72.09139,90 90,72.09139 90,50 C90,27.90861 72.09139,10 50,10 C27.90861,10 10,27.90861 10,50" id="Oval" stroke="#FFF" stroke-width="4" transform="translate(50.000000, 50.000000) rotate(16.000000) translate(-50.000000, -50.000000) "></path>
                <path d="M50.1069716,80.1069716 C66.675514,80.1069716 80.1069716,66.675514 80.1069716,50.1069716 C80.1069716,33.5384291 66.675514,20.1069716 50.1069716,20.1069716 C33.5384291,20.1069716 20.1069716,33.5384291 20.1069716,50.1069716" id="Oval-Copy-2" stroke="#FFF" stroke-width="4" transform="translate(50.106972, 50.106972) rotate(40.000000) translate(-50.106972, -50.106972) "></path>
                <path d="M57.6870117,59.3168945 C59.2251053,60.3764701 60.6435482,61.151202 61.9423828,61.6411133 L60.9682617,63.9482422 C59.1681225,63.2988249 57.3737069,62.2734445 55.5849609,60.8720703 C53.7278553,61.9088594 51.6770946,62.4272461 49.4326172,62.4272461 C47.1653532,62.4272461 45.1088959,61.8803766 43.2631836,60.7866211 C41.4174712,59.6928656 39.9961801,58.1547951 38.9992676,56.1723633 C38.002355,54.1899315 37.5039062,51.9568809 37.5039062,49.4731445 C37.5039062,47.0008014 38.0052033,44.7506612 39.0078125,42.7226562 C40.0104217,40.6946513 41.4374094,39.1508842 43.2888184,38.0913086 C45.1402274,37.031733 47.210926,36.5019531 49.5009766,36.5019531 C51.8138136,36.5019531 53.8987537,37.0516709 55.7558594,38.151123 C57.612965,39.2505752 59.0285596,40.7886457 60.0026855,42.7653809 C60.9768115,44.742116 61.4638672,46.9723183 61.4638672,49.4560547 C61.4638672,51.5182395 61.1505565,53.372469 60.5239258,55.0187988 C59.897295,56.6651287 58.9516665,58.0978129 57.6870117,59.3168945 Z M50.3896484,55.0786133 C52.3037205,55.6140977 53.881667,56.4116158 55.1235352,57.4711914 C57.0717871,55.6938388 58.0458984,53.0221532 58.0458984,49.4560547 C58.0458984,47.4280498 57.7012567,45.6564203 57.0119629,44.1411133 C56.3226691,42.6258062 55.3143784,41.4494671 53.9870605,40.6120605 C52.6597427,39.774654 51.1700929,39.355957 49.5180664,39.355957 C47.0457233,39.355957 44.9949626,40.2018958 43.3657227,41.8937988 C41.7364827,43.5857018 40.921875,46.1121251 40.921875,49.4731445 C40.921875,52.7316244 41.7279379,55.2324132 43.3400879,56.9755859 C44.9522379,58.7187587 47.0115435,59.590332 49.5180664,59.590332 C50.7029682,59.590332 51.8194935,59.3681663 52.8676758,58.9238281 C51.8308867,58.2516242 50.7371477,57.7731134 49.5864258,57.4882812 L50.3896484,55.0786133 Z" id="Q" fill="#49c7da"></path>
            </g>
        </g>
    </g>
</svg>