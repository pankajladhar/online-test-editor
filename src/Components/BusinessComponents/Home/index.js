import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';

class Home extends PureComponent {
    render() {
        return (
            <div className="Home">
                
            </div>
        );
    }
}

Home.propTypes = {

};

export default Home;