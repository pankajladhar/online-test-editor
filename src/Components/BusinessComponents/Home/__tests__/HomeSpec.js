import React from 'react';
import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router-dom'
import Home from './../index';

describe('Home', ()=>{
    it('renders correctly', ()=>{
        let tree = renderer.create(
            <StaticRouter location="someLocation" context={{}}>
                <Home/>
            </StaticRouter>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});