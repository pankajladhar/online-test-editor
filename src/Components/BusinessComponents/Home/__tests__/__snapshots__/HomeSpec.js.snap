// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="Home"
>
  <h1>
    Quizbox - Online quiz builder
  </h1>
  <h2>
    Build your own quiz and share with others
  </h2>
  <a
    href="/configure"
    onClick={[Function]}
  >
    Configure Quiz
  </a>
</div>
`;
